bitnami/mongodb/templates/_helpers.tpl

```diff
@@ -6,6 +6,31 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Allow the release namespace to be overridden for multi-namespace deployments in combined charts.
+*/}}
+{{- define "mongodb.namespace" -}}
```

Comment:
Following the approach of the rest of globals, the global should have preference over the rest of parameters

Reply:
I can make this change if you want, but is that the most useful behaviour? If the local takes preference then a chart can still choose a specific namespace for its child when its own global namespace is overridden. If you switch the priority round then you force all dependencies of dependencies into a single namespace

Reply:
to fix this issue, I ended up not having local variable for other charts, for example: [mongodb-replicaset](https://github.com/helm/charts/blob/66f56ec2903cfdd45479e4835e58fcfaa5238291/stable/mongodb-replicaset/templates/_helpers.tpl#L80-L93) as global variables still can be set in local values file,

Reply:
Ah, okay, that makes more sense now, thanks 😄